service/tbc/tbc.go

```diff
@@ -2641,10 +2658,11 @@ func (s *Server) ExternalHeaderSetup(ctx context.Context, upstreamStateId []byte
 	defer log.Tracef("ExternalHeaderSetup exit")
 
 	if !s.cfg.ExternalHeaderMode {
-		return errors.New("ExternalHeaderSetup called but external header mode is not enabled in config")
+		return errors.New("ExternalHeaderSetup called but external" +
```

Comment:
There's a missing space here, so the error would be "...externalheader mode..."
```suggestion
		return errors.New("ExternalHeaderSetup called but external " +
```

Reply:
Fixed